api/c/indigo/src/indigo_molecule.cpp

```diff
@@ -3784,6 +3784,30 @@ CEXPORT int indigoCountComponentBonds(int molecule, int index)
     INDIGO_END(-1);
 }
 
+CEXPORT const char* indigoGetOriginalFormat(int item)
+{
+    INDIGO_BEGIN
+    {
+        IndigoObject& obj = self.getObject(item);
+        if (IndigoBaseMolecule::is(obj))
+        {
+            BaseMolecule& mol = obj.getBaseMolecule();
```

Comment:
make original format to return correct content types, e.g. "chemical/x-daylight-smarts